src/Microsoft.ML.TimeSeries/TimeSeriesStatic.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>;
+
+namespace Microsoft.ML.StaticPipe
+{
+    /// <summary>
+    /// IidChangePoint static API extension methods.
+    /// </summary>
+    public static class IidChangePointStaticExtensions
+    {
+        public class Prediction
+        {
+            public int Alert;
+            public float Score;
+            public float PValue;
+            public float MartingaleValue;
+        }
+
+        private sealed class OutColumn : Vector<Prediction>
```

Comment:
@sfilipi Is it the right thing to do here to create a custom Prediction class specifies what my output column should look like?